pkg/controller/operands/monitoring.go

```diff
@@ -249,17 +250,30 @@ func NewPrometheusRuleSpec() *monitoringv1.PrometheusRuleSpec {
 	return &monitoringv1.PrometheusRuleSpec{
 		Groups: []monitoringv1.RuleGroup{{
 			Name: alertRuleGroup,
-			Rules: []monitoringv1.Rule{{
-				Alert: outOfBandUpdateAlert,
-				Expr:  intstr.FromString("sum by(component_name) ((round(increase(kubevirt_hco_out_of_band_modifications_count[10m]))>0 and kubevirt_hco_out_of_band_modifications_count offset 10m) or (kubevirt_hco_out_of_band_modifications_count != 0 unless kubevirt_hco_out_of_band_modifications_count offset 10m))"),
-				Annotations: map[string]string{
-					"description": "Out-of-band modification for {{ $labels.component_name }} .",
-					"summary":     "{{ $value }} out-of-band CR modifications were detected in the last 10 minutes.",
+			Rules: []monitoringv1.Rule{
+				{
+					Alert: outOfBandUpdateAlert,
+					Expr:  intstr.FromString("sum by(component_name) ((round(increase(kubevirt_hco_out_of_band_modifications_count[10m]))>0 and kubevirt_hco_out_of_band_modifications_count offset 10m) or (kubevirt_hco_out_of_band_modifications_count != 0 unless kubevirt_hco_out_of_band_modifications_count offset 10m))"),
+					Annotations: map[string]string{
+						"description": "Out-of-band modification for {{ $labels.component_name }}.",
+						"summary":     "{{ $value }} out-of-band CR modifications were detected in the last 10 minutes.",
+					},
+					Labels: map[string]string{
+						"severity": "warning",
+					},
 				},
-				Labels: map[string]string{
-					"severity": "warning",
+				{
+					Alert: unsafeModificationAlert,
+					Expr:  intstr.FromString("sum by(annotation_name) (max_over_time(kubevirt_hco_unsafe_modification_count[10m])>0)"),
```

Comment:
When you use `max_over_time` instead of `increase`,  what is the purpose of 10minutes window? This will be true forever if the counter is more than zero. As far as I remember, the decision was to keep the alert in FIRING state for a while. If you think it is just `info` and we can keep it in FIRING state as long as it is more than zero, please just change the query and check whether it is more than zero or not. Otherwise, please use `increase` to respect deltas instead of status.

Reply:
/unhold